src/LinuxTracing/PerfEvent.h

```diff
@@ -35,7 +35,7 @@ perf_event_sample_regs_user_all_to_register_array(const perf_event_sample_regs_u
 // while the `data` field holds the data specific to the individual event.
 template <typename PerfEventDataT>
 struct TypedPerfEvent {
-  uint64_t timestamp;
+  uint64_t timestamp{};
```

Comment:
Personal preference across the change: For member variables I prefer explicit values (e.g., `0`, `nullptr`, `false`). Also the next field is initialized more explicitly.

Reply:
Agree. Changed it here and hopefully across the pr.